app/questionnaire/location.py

```diff
@@ -0,0 +1,15 @@
+class Location(object):
+
+    def __init__(self, group_id, group_instance, block_id):
+
+        self.group_id = group_id
+        self.group_instance = group_instance
+        self.block_id = block_id
+
+    def __eq__(self, other):
```

Comment:
Usually best to also define `__hash__` when defining `__eq__` e.g. http://stackoverflow.com/questions/1608842/types-that-define-eq-are-unhashable